app/models.py

```diff
@@ -448,6 +468,7 @@ def serialize(self):
             "billing_contact_email_addresses": self.billing_contact_email_addresses,
             "billing_reference": self.billing_reference,
             "can_approve_own_go_live_requests": self.can_approve_own_go_live_requests,
+            "organisation_permissions": [x.permission for x in self.permissions],
```

Comment:
Can we just call this `permissions`? Having the `organisation_` prefix is still redundant